docs/src/understand/single-sign-on/trouble-shooting.md

```diff
@@ -1,88 +1,174 @@
 (trouble-shooting-faq)=
 
+```{contents}
+:depth: 2
+```
```

Comment:
@arthurwolf you have used the same idiom in https://raw.githubusercontent.com/wireapp/wire-server/develop/docs/src/understand/single-sign-on/understand/main.md with much better success.  what am i doing wrong here?

(it's not important, only solve it if it's easy, otherwise we can keep it as it is now.  thanks!)